src/test-fbp/converter-error-expected.txt

```diff
@@ -0,0 +1 @@
+WRN: ./src/lib/common/sol-types.c:242 sol_irange_division() Division by zero: 10, 0
```

Comment:
The problem is that this warning depends on the line number of the code it is in. If someone adds something to sol-types.c completely unrelated, this will change.

Maybe what's missing here is that we need to tweak the log level that those tests work (in `src/test-fbp/run`), since this is not what we expect to production users to see unless debug is enable (and we don't want to test for output of debug messages).

What do you think?